drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -320,10 +322,13 @@ def _learner_raw_score(self):
         return correct_count / float(total_count)
 
     @staticmethod
-    def _get_statici18n_js_url():
+    def get_deprecated_i18n_js_url():
```

Comment:
Nit: this method should stay private (i.e., have the `_` prefix).

Reply:
Can static methods be also private? I've made it static and private and and changed the call from `DragAndDropBlock.get_deprecated_i18n_js_url` --> `self._get_deprecated_i18n_js_url`.